modules/debugging/rendering/screenspacedebugplane.cpp

```diff
@@ -34,37 +34,34 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
-    struct [[codegen::Dictionary(RenderableDebugPlane)]] Parameters {
+    // This `ScreenSpaceRenderable` can be used for debugging OpenGL textures. It renders
+    // the content of an existing texture, based on a provided OpenGL texture name.
+    struct [[codegen::Dictionary(ScreenSpaceDebugPlane)]] Parameters {
         // [[codegen::verbatim(TextureInfo.description)]]
         std::optional<int> texture;
     };
-#include "renderabledebugplane_codegen.cpp"
+#include "screenspacedebugplane_codegen.cpp"
 } // namespace
 
 namespace openspace {
 
-documentation::Documentation RenderableDebugPlane::Documentation() {
-    return codegen::doc<Parameters>(
-        "debugging_renderable_debugplane",
-        RenderablePlane::Documentation()
-    );
+documentation::Documentation ScreenSpaceDebugPlane::Documentation() {
+    return codegen::doc<Parameters>("debugging_screenspace_debugplane");
 }
 
-RenderableDebugPlane::RenderableDebugPlane(const ghoul::Dictionary& dictionary)
-    : RenderablePlane(dictionary)
+ScreenSpaceDebugPlane::ScreenSpaceDebugPlane(const ghoul::Dictionary& dictionary)
+    : ScreenSpaceRenderable(dictionary)
     , _texture(TextureInfo, -1, -1, 512)
```

Comment:
```suggestion
    , _texture(TextureInfo, -1, -1, 4096)
```

Should probably increase that since we already are using more that 512 textures